### ISSUE TITLE
Relative links validation failure for internal links

### ISSUE THREAD

**Issue by atiq-cs** [statiqdev/Statiq#954]
In this case, I am getting validation failure for `tags/podcasts`.  I do have the tag: podcasts. However, it is still failing validation and it is generating lot of warnings.

    blog/input/posts/celebrating-science-day.md => posts/celebrating-science-day.html] Could not validate relative link: <a class="nav-link" href="tags/podcasts">পডকাস্ট</a> (ValidateRelativeLinks)
    ... ... ...
    same error continues for all the posts..
    ... ... ....
    [WARN] 59 total relative links could not be validated (ValidateRelativeLinks)

**Comment by daveaglick** [comment]
Sorry for the _extremely_ late reply 😬. I think some of the work done since this was filed may have resolved it. Are you still seeing the problem in the most recent releases?

**Comment by RohitM-IN** [comment]
I am still seeing this in Statiq.Docs   Version="1.0.0-beta.15"

**Comment by daveaglick** [comment]
Any chance you could post/send a reproducible example, @RohitM-IN? Since this issue was originally opened and my reply, there was a pretty big rework of the link validation code that should have cleared things up. If you're seeing issues it may not be the same as what was originally reported here.

**Comment by RohitM-IN** [comment]
> If you're seeing issues it may not be the same as what was originally reported here.

yes I am sorry I meant was some of the relative links was not generating and most of them was generic class/methods API links 

I was searching that time for the relative links issue now opened up discussion 

I was talking about statiqdev/discussions/210
